### PR TITLE
Add a bit of README docs

### PR DESCRIPTION
**Pull request by marians**
For https://github.com/giantswarm/giantswarm/issues/4347 but only cosmetic changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+### AWS
+
+In AWS, this cleans up:
+
+- CloudFormation stacks
+  - that are older than 90 minutes
+  - matching certain name prefixes (`cluster-ci-`, `host-peer-ci-`, `e2e-`)
+- S3 buckets
+  - that are older than 90 minutes
+  - matching certain name criteria
```
Comment (JosephSalisbury) [inline README.md]: What are those criteria?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/ci-cleaner.svg?style=shield)](https://circleci.com/gh/giantswarm/ci-cleaner)
+[![CircleCI](https://circleci.com/gh/giantswarm/ci-cleaner/tree/master.svg?style=svg&circle-token=200804d99fdd5ee97482012f23d4470b62f8e34c)](https://circleci.com/gh/giantswarm/ci-cleaner/tree/master)
 
-# aws-ci-cleaner
+# CI Cleaner
+
+Cleans up cloud provider resources created during tests in CI (continuous integration)
```
Comment (JosephSalisbury) [inline README.md]: A full stop at the end here would be top notch

File: README.md (hunk 1 above)
Comment (marians) [reply README.md]: It's not as simple as in the case above, so I would recommend to look at the source.

**Comment by marians** [comment]
@JosephSalisbury mergeable?

**Review by JosephSalisbury — APPROVED** [review]
Very mergeable <3

**Pull request merged**